src/BuildingRegistry/Building/Events/BuildingWasMerged.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace BuildingRegistry.Building.Events
+{
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using Be.Vlaanderen.Basisregisters.EventHandling;
+    using Be.Vlaanderen.Basisregisters.GrAr.Common;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Newtonsoft.Json;
+
+    [EventTags(EventTag.For.Sync, EventTag.For.Edit, Tag.Building)]
+    [EventName(EventName)]
+    [EventDescription("Het gebouw werd samengevoegd in het doelgebouw.")]
+    public sealed class BuildingWasMerged : IBuildingEvent
```

Comment:
This event is also defined in PR 1070 where it has different property names.

Reply:
Removed from PR #1070, didn't belong there.